src/components/LandingPage/LandingPage.scss

```
@@ -0,0 +1,80 @@
+@use "../../spacing" as *;
+@use "../../font-size" as *;
+@import "../../colours.scss";
+@import "../../mixins.scss";
+
+.landing-page-wrapper {
+  margin: 0 auto;
+  padding-bottom: $space-1;
+  text-align: center;
+}
+
+// Projects
+
+.landing-page__projects {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  @mixin unique-width($width) {
+    @media screen and (min-width: $width) {
+      @content;
+    }
+  }
+}
+
+.landing-page__projects--title {
+  @include font-size-3(bold);
+  margin: $space-1;
+}
+
+.landing-page__projects--subtitle {
+  @include font-size-2(regular);
+  font-weight: var(--font-weight-regular);
+  margin: $space-1;
+}
+
+.landing-page__projects--login {
+  @include font-size-1-5(regular);
```

fonts a little too big at the minute

```suggestion
  @include font-size-1-25(regular);
```